src/haddock/libs/libio.py

```diff
@@ -35,6 +37,101 @@ def read_from_yaml(yaml_file):
     return ycfg
 
 
+def write_dic_to_file(
+        data_dict,
+        output_fname,
+        info_header="",
+        sep="\t",
+        ):
+    """
+    Write a dictionary to a file.
+
+    Parameters
+    ----------
+    data_dict : dict
+        Dictionary to write.
+    output_fname : str or Path
+        Name of the output file.
+    info_header : str
+        Header to write before the data.
+    """
+    with open(output_fname, 'w') as fout:
+        fout.write(info_header)
+        fout.write(str(data_dict))
```

Comment:
Here. You might want to use `json.dumps` instead of `str`.

Reply:
?